docs/scenarios_authoring/RRDataSourcesAndSinks.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 4
+---
+
+# Request-Response sources and sinks
+
+## Http request as input and output to Nussknacker scenarios
+
+### Concepts
+
+In Request-response mode you interact with Nussknacker Lite runtime engine in a similar way as with any "server" in the
```

Comment:
Maybe we can add that each scenario exposes REST API (and then "if you don't know this terminology, ...)

Reply:
Can elaborate on that ? This is an exact "user friendly" description Zbyszek gave me ;) I feat that if we introduce REST API then we should explain what REST API is :)